about.css

```diff
@@ -0,0 +1,661 @@
+    margin-top: 0;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .menu_list li {
+    margin-bottom: 5%;
+  }
+
+  .showMenu {
+    transform: translateY(0);
+    display: block;
+  }
+
+  .paragraph {
+    border: 4px solid #ececec;
+    padding: 20px 10px;
+    color: #7c7c7c;
+    background-color: white;
+  }
+
+  .message {
+    color: #7c7c7c;
+    text-align: center;
+  }
+
+  .place {
+    text-decoration: underline;
+  }
+
+  .banner {
+    padding-bottom: 12%;
+    text-align: center;
+  }
+
+  .partners {
+    background-color: #424146;
+    color: white;
+    text-align: center;
+    padding-top: 3%;
+    padding-bottom: 3%;
+    padding-left: 10%;
+    padding-right: 10%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .partener_logos {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .partener_logos li {
+    width: 30%;
+  }
+
+  header {
+    background-image: url("./pictures/main_big.png");
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    padding: 20px;
+  }
+
+  .programs {
+    background-color: white;
+    padding: 20px;
+    padding-bottom: 5%;
+  }
+
+  .red_bar {
+    border: 1px solid #9c3e36;
+    width: 10%;
+    margin-bottom: 5%;
+  }
+
+  .card_program {
+    width: 60%;
+    height: 20%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #f1f1f1;
+  }
+
+  .mobile_footer {
+    padding-top: 3%;
+    padding-bottom: 3%;
+    padding-left: 10%;
+    padding-right: 10%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .footer_text {
+    width: 60%;
+  }
+
+  .footer_text p {
+    margin-top: 0;
+    color: #78787a;
+  }
+
+  .logoLionFooter {
+    width: 20%;
+  }
+
+  .centerLogo {
+    width: 40%;
+  }
+
+  .card_program h3 {
+    margin-left: 20px;
+    margin-right: 20px;
+    color: #c74d38;
+  }
+
+  .join {
+    background-color: #ff4f38;
+    margin: 0 auto;
+    padding: 20px;
+    border: 0;
+    margin-top: 6%;
+    margin-bottom: 3%;
+    color: white;
+    cursor: pointer;
+    width: 80%;
+  }
+
+  .action {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .title {
+    width: 100%;
+    padding-top: 8%;
+    margin-top: 0;
+    text-align: center;
+  }
+
+  .red_bar2 {
+    border: 1px solid #fda69c;
+    width: 10%;
+    margin-bottom: 5%;
+  }
+
+  .speakers {
+    background-color: #f8f8f8;
+    border-top: 1px solid #f0f0f0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-left: 5%;
+    padding-right: 5%;
+  }
+
+  .aboutCard1 {
+    text-align: center;
+    padding: 20px;
+    color: white;
+    overflow: hidden;
+    background: #c04848;
+    background: linear-gradient(rgb(72, 0, 72, 0.8), rgb(192, 72, 72, 0.8)), url("/pictures/2016-past.jpg");
```

Comment:
The overlay look so cool. But kindly use color swatch defined on the design guide. That's the orange #ec5242 value. We need to follow colors, fonts, spacings for design, only content can be changed.

Reply:
@huseyinbiyik  my last comment is they didn't say to follow 100% the same website please read the project requirements.

Reply:
@Yacoubou-seidou, kindly check the project requirements.
<img width="1016" alt="Screenshot 2023-03-14 at 11 24 11 PM" src="https://user-images.githubusercontent.com/25887253/225128051-d255a0a7-c8f1-4138-87ee-c6625538e858.png">